growthviz/sumstats.py

```diff
@@ -170,6 +170,8 @@ def bmi_stats(
     if out == None:
         return merged_stats
     else:
+        with out:
```

Comment:
@mmedle I'm a little confused about this section. It looks like it might have been the `wait=False` bit that we needed?

- do we need the `with` block?
- should the second line be `out.clear_output(wait=False)`? the function `clear_output()` isn't in the namespace otherwise - see https://github.com/mitre/GrowthViz/actions/runs/3606042921/jobs/6076999791#step:5:19

I can follow up on this, just wanted to check to see if you'd meant to add `out.clear_output(...)` instead of just `clear_output(...)`.

Thanks!

Reply:
Nevermind - following up on your changes, I searched some more and found a slightly different approach that seems to resolve everything!